### PR TITLE
add linkerd uses iptables blog

### PR DESCRIPTION
**Pull request by C-Mejlak**
Replaces #1190 and #1192

### CODE REVIEW

**Comment by wmorgan** [comment]
It's late in the game for wordsmithing, but looking at this again, IMO the title would read better as "How Linkerd uses iptables to tranparently route Kubernetes traffic". @mateiidavid wdyt?

**Comment by mateiidavid** [comment]
> It's late in the game for wordsmithing, but looking at this again, IMO the title would read better as "How Linkerd uses iptables to tranparently route Kubernetes traffic". @mateiidavid wdyt?

That sounds good to me!

File: linkerd.io/content/blog/linkerd-uses-iptables-to-route-kubernetes-traffic-transparently.md [hunk 1]
```diff
@@ -0,0 +1,260 @@
+---
+title: How Linkerd uses iptables to route Kubernetes traffic transparently
```
Comment (mateiidavid) [inline linkerd.io/content/blog/linkerd-uses-iptables-to-route-kubernetes-traffic-transparently.md:2]:
```suggestion
title: How Linkerd uses iptables to transparently route Kubernetes traffic
```

Comment (mateiidavid) [reply linkerd.io/content/blog/linkerd-uses-iptables-to-route-kubernetes-traffic-transparently.md:2]: oh can't automagically commit from the UI, don't have write perms on the fork :(

**Review by mateiidavid — APPROVED** [review]
🚢 📦 if the title can be changed

**Comment by C-Mejlak** [comment]
Title updated

**Comment by mateiidavid** [comment]
Thank you @C-Mejlak! 🎊

**Pull request merged**